05/app.js

```diff
@@ -13,12 +14,11 @@ class Weather extends React.Component {
     super(props);
 
     this.apiKey = '1b4ea2ecd37c4174ac406ce8a2376027';
+    this.api = new API();
```

Comment:
Skoro tutaj tworzysz obiekt (i dobrze, tak jest lepiej) to powinnaś wykorzystać tą metodę.


PS. Pamiętaj, aby przekazać URL. Dane o długości i szerokości geogr. powinny być przekazana przez `props`, nie trzeba ich zapisywać w `state` czyli: ``` `https://api.weatherbit.io/v2.0/current?lat=${props.latitude}&lon=${props.longitude}&key=${this.apiKey}&units=M&lang=pl` ```